docs/spec/rum_v3_context.json

```diff
@@ -0,0 +1,298 @@
+            },
+            "required": ["url", "mt"]
+        },
+        "g": {
+            "$ref": "tags.json"
+        },
+        "u": {
+            "description": "Describes the correlated user for this event. If user data are provided here, all user related information from metadata is ignored, otherwise the metadata's user information will be stored with the event.",
+            "properties": {
+                "id": {
+                    "description": "Identifier of the logged in user, e.g. the primary key of the user",
+                    "type": ["string", "integer", "null"],
+                    "maxLength": 1024
+                },
+                "em": {
+                    "description": "Email of the logged in user",
+                    "type": ["string", "null"],
+                    "maxLength": 1024
+                },
+                "un": {
+                    "description": "The username of the logged in user",
+                    "type": ["string", "null"],
+                    "maxLength": 1024
+                }
+            }
+        },
+        "p": {
+            "description": "",
+            "type": ["object", "null"],
+            "properties": {
+                "rf": {
+                    "description": "RUM specific field that stores the URL of the page that 'linked' to the current page.",
+                    "type": ["string", "null"]
+                },
+                "url": {
+                    "description": "RUM specific field that stores the URL of the current page",
+                    "type": ["string", "null"]
+                }
+            }
+        },
+        "se": {
+            "description": "Service related information can be sent per event. Provided information will override the more generic information from metadata, non provided fields will be set according to the metadata information.",
+            "properties": {
+                "a": {
+                    "description": "Name and version of the Elastic APM agent",
+                    "type": ["object", "null"],
+                    "properties": {
+                        "n": {
+                            "description": "Name of the Elastic APM agent, e.g. \"Python\"",
+                            "type": ["string", "null"],
+                            "maxLength": 1024
+                        },
+                        "ve": {
+                            "description": "Version of the Elastic APM agent, e.g.\"1.0.0\"",
+                            "type": ["string", "null"],
+                            "maxLength": 1024
+                        },
+                        "ephemeral_id": {
+                            "description": "Free format ID used for metrics correlation by some agents",
+                            "type": ["string", "null"],
+                            "maxLength": 1024
+                        }
+                    }
+                },
+                "framework": {
```

Comment:
we might start sending these field since we already have framework specific information. The reason why they are not sent right now its mainly bcoz of payload size. Can we also shorten these fields